mlir/include/mlir/Dialect/Rock/IR/RockOps.td

```diff
@@ -1190,9 +1219,13 @@ def Rock_BlockwiseBroadcastReduceOp:
 
     $workspace_buffer resides in LDS. This should be a flat tensor with at least
     number of elements in the input tensor.
+
```

Comment:
This documentation doesn't make it particularly clear what this extra parameter _does_.

Reply:
ok Ive changed the wording...